src/index.css

```
@@ -131,17 +131,25 @@ body {
+
 .start-btn {
   background: var(--accent-color);
   color: white;
   border: none;
-  padding: 1.25rem 2.5rem;
-  font-size: 1.25rem;
+  padding: 1rem 2.5rem;
+  font-size: 1.1rem;
   border-radius: 16px;
   cursor: pointer;
   font-family: 'Outfit', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)